contracts/tokens/SecurityToken.sol

```diff
@@ -341,13 +341,16 @@ contract SecurityToken is StandardToken, DetailedERC20, ReentrancyGuard, Registr
         return modules[_type];
     }
 
-    /**
-    * @notice allows the owner to withdraw unspent POLY stored by them on the ST.
+   /**
+    * @notice allows the owner to withdraw unspent POLY stored by them on the ST or any ERC20 token.
     * @dev Owner can transfer POLY to the ST which will be used to pay for modules that require a POLY fee.
+    * @param _tokenContract Address of the ERC20Basic compliance token
     * @param _value amount of POLY to withdraw
     */
-    function withdrawPoly(uint256 _value) external onlyOwner {
-        require(ERC20(polyToken).transfer(owner, _value), "Insufficient balance");
+    function withdrawERC20(address _tokenContract, uint256 _value) external onlyOwner {
+        require(_tokenContract != address(0));
+        IERC20 token = IERC20(_tokenContract);
+        require(token.transfer(owner, _value));
```

Comment:
Missing reason strings

Reply:
It is intentional to reduce the size of the contract